ironic_prometheus_exporter/messaging.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+from oslo_config import cfg
+from oslo_messaging.notify import notifier
+
+prometheus_opts = [
+    cfg.StrOpt('file_name', required=True),
+    cfg.StrOpt('file_dir', required=True)
+]
+
+
+def register_opts(conf):
+    conf.register_opts(prometheus_opts, group='oslo_messaging_notifications')
+
+
+class PrometheusFileDriver(notifier.Driver):
+
+    "Publish notifications into a File to be used by Prometheus"
```

Comment:
Nit: This won't function as a docstring. It should use the """ notation.

"""Publish notifications into a File to be used by Prometheus"""

Reply:
ack